modules/order/src/Entity/OrderInterface.php

```diff
@@ -222,4 +223,23 @@ public function getBillingProfileId();
    */
   public function setBillingProfileId($billingProfileId);
 
+  /**
+   * Adds an adjustment to the order.
+   *
+   * @param \Drupal\commerce_order\Adjustment $adjustment
+   *    The adjustment.
+   *
+   * @return $this
+   *
+   * @todo: How do we remove or clear them?
+   */
+  public function addAdjustment(Adjustment $adjustment);
+
+  /**
+   * Causes the order to recalculate its total.
+   *
+   * This will update the total price based on line item and adjustment prices.
+   */
+  public function calculateTotal();
```

Comment:
Doesn't feel like this should be a public interface method to me. It's more of an internal helper.